### ISSUE TITLE
Move test code away from live code

### ISSUE THREAD

**Issue by subotic** [dhlab-basel/Knora#359]
Inside ``ResourcesRouteV1.scala``:

 - ``path("v1" / "error" / Segment)``

Inside ``ResourcesResponderV1.scala``:

 - ``case UnexpectedMessageRequest() => ``
 - ``case InternalServerExceptionMessageRequest() => ``

**Comment by benjamingeer** [comment]
I was thinking we could make an `ErrorTestResponderV1` for this, would that be OK?

**Comment by subotic** [comment]
Yes, I was thinking the same thing. However, this responder should "live" inside the test directory, since it will only be used inside tests.
